File: selector/src/main/java/com/luck/picture/lib/interfaces/OnQueryFilterListener.java
package com.luck.picture.lib.interfaces;

import com.luck.picture.lib.entity.LocalMedia;

/**
 * @author：luck
 * @date：2022/3/12 10:09 上午
 * @describe：OnQueryFilterListener
 */
public interface OnQueryFilterListener {
    /**
     * You need to filter out what doesn't meet the standards
     *
     * @return the boolean result
     */
    boolean onFilter(LocalMedia media);
}

File: selector/src/main/java/com/luck/picture/lib/interfaces/OnQueryFilterListener.kt
package com.luck.picture.lib.interfaces

import com.luck.picture.lib.entity.LocalMedia

/**
 * @author：luck
 * @date：2020-01-14 17:08
 * @describe：Filter Listener
 */
interface OnQueryFilterListener {
    /**
     * Filter out multimedia data that does not comply with rules
     * @param media media information
     * @return true filtering
     */
    fun onFilter(media: LocalMedia): Boolean
}